index.d.ts

```diff
@@ -0,0 +1,40 @@
+declare module "ember-cookies/services/cookies" {
```

Comment:
@And012 as this is monorepo, you should include those in `packages/ember-cookies`